build-tools/debian-metadata/xamarin.android.install

```diff
@@ -0,0 +1,2 @@
+bin/Debug/bin/xabuild		/usr/bin
```

Comment:
We may want to instead consider shipping `bin/Release`, not `bin/Debug`.

The question is: do we want to ship binaries full of debugging symbols? For comparison, `Mono.Android.dll` is ~2.5MB smaller in the Release build.